src/docs/cmdstan-guide/installation.tex

```diff
@@ -618,12 +620,17 @@ \section{Optional Parallelization Support}
     different machines. Please refer to
     \url{https://github.com/stan-dev/math/wiki/MPI-Parallelism} for
     more details.
+  \item[OpenCL] in \CmdStan enables Cholesky decompositions to utilize a GPU
+   for large parallel computations. Please refer to
+   \url{https://github.com/stan-dev/math/wiki/OpenCL-GPU-Routines} for setup
```

Comment:
One issue with this. The install instructions we have on the wiki are tested on Ubuntu 18.04, Mac, and Windows. But if feel like as people start using this we will need to update those with problems people report and find resolutions for. I'd rather keep the install instructions on the wiki so that they are easier to modify. Does that make sense?